Document/0x05d-Testing-Data-Storage.md

```diff
@@ -1069,22 +1069,35 @@ setContentView(R.layout.activity_main);
 
 #### Overview
 
-Analyzing the memory can help to identify the root cause of different problems, like for example why an application is crashing, but can also be used to identify sensitive data. This section describes how to check for sensitive data and disclosure of data in general within the process memory.
+Analyzing memory can help developers to identify root causes of different problems, like for example application crashes, etc. However, it can also be used to gain access to sensitive data. This section describes how to check for disclosure of data within the process' memory.
```

Comment:
> like for example application crashes, etc. 

This should be "such as application crashes". I think "like", "for example" and "etc" is a bit too much to indicate this is just an example.

> different problems

Shouldn't this be "several problems"?